drivers/volume/portworx/portworx.go

```diff
@@ -428,6 +435,41 @@ func (p *portworx) SnapshotCreate(
 		return nil, getErrorSnapshotConditions(err), err
 	}
 
+	backgroundCommandTermChan := make(chan bool, 1)
```

Comment:
The driver shouldn't know about rules and how they are configured. All they should do is
chan, err = rule.PreSnapRun(snapshot)
// trigger snapshot and signal channel when snapshot is taken
rule.PostSnapRun(snapshot)